assets/admin/editor-wizard/steps/upgrade-step.js

```diff
@@ -1,25 +1,52 @@
+/**
+ * WordPress dependencies
+ */
+import { Button } from '@wordpress/components';
+
 /**
  * Upgrade step during course creation wizard.
  */
 const UpgradeStep = () => {
-	// TODO Implement this.
 	return (
-		<div>
-			<div>PENDING TO IMPLEMENT</div>
+		<div className="components-modal-columns">
+			<div className="components-modal-columns__description">
+				<p>
+					Do you want to sell this course? This requires Sensei Pro
+					which also unlocks many useful features.
+				</p>
+				<h2>$149 USD</h2>
+				<span className="label">per year, 1 site</span>
+				<ul>
+					<li>WooCommerce integration</li>
+					<li>Schedule &apos;drip&apos; content</li>
+					<li>Set expiration date of courses</li>
+					<li>Quiz timer</li>
+					<li>Flashcards, Image Hotspots, and CHecklists</li>
+					<li>1 year of updates & support</li>
+				</ul>
+			</div>
+			<div className="components-modal-columns__image">
+				<p>Image container</p>
+			</div>
 		</div>
 	);
 };
 
-UpgradeStep.Title = 'Upgrade Step';
+UpgradeStep.Title = 'Sell with Sensei Pro';
```

Comment:
I'd recommend as part of this PR a refactor to the title. We can just not send a title to the `<Modal>`, and add the title as part of the content in the steps. Here's an example that we have of a modal without a title in Gutenberg: https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/guide/index.js

Reply:
Done in the commits [e470151](https://github.com/Automattic/sensei/pull/5169/commits/e470151863acfd8d69b12cbcf7be6c5a2a01a1db) and [bff4106](https://github.com/Automattic/sensei/pull/5169/commits/bff4106c9fb5fe5b23a1c89389c5a68abaca144a) (and I added you as a co-author after our pair programming session, hehe)